### PR TITLE
SONAR-8172 make all arguments of api/organizations/update optional

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

File: sonar-plugin-api/src/main/java/org/sonar/api/server/ws/Request.java [hunk 1]
```diff
@@ -304,4 +335,217 @@ private static long parseLong(String key, String value) {
...
+   * Implementation of {@link Param} where the param is present.
+   */
+  private static final class GenericParam<T> implements Param<T> {
+    private final T value;
+
+    private GenericParam(T value) {
+      this.value = value;
+    }
+
+    protected static <T> Param<T> present(T value) {
```
Comment (SonarTech) [inline sonar-plugin-api/src/main/java/org/sonar/api/server/ws/Request.java]: MAJOR Remove this "protected" modifier. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2156)

File: sonar-plugin-api/src/main/java/org/sonar/api/server/ws/Request.java [hunk 2]
```diff
@@ -304,4 +335,217 @@ private static long parseLong(String key, String value) {
...
+    @CheckForNull
+    private final String value;
+    private final boolean emptyAsNull;
+
+    private StringParamImpl(@Nullable String value, boolean emptyAsNull) {
+      this.value = value;
+      this.emptyAsNull = emptyAsNull;
+    }
+
+    protected static StringParam present(String value) {
```
Comment (SonarTech) [inline sonar-plugin-api/src/main/java/org/sonar/api/server/ws/Request.java]: MAJOR Remove this "protected" modifier. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2156)

**Comment by SonarTech** [comment]
SonarQube analysis reported 3 issues
* MAJOR 3 major

Watch the comments in this conversation to review them.

**Comment by sns-seb** [comment]
merged manually

**Pull request closed without merging**